### ISSUE TITLE
support for aria-braille* properties

### ISSUE THREAD

**Issue by pkra** [FreedomScientific/VFO-standards-support#677]
The ARIA Editors' Draft proposes the aria-braillelabel and aria-brailleroledescription property, see also https://w3c.github.io/aria/#aria-braillelabel, https://w3c.github.io/aria/#aria-brailleroledescription.

These properties are on track for ARIA 1.3. Browser support includes Gecko, WebKit and Chromium. (See also the 1.3 overview issue https://github.com/w3c/aria/issues/1331 for more links to various bits and pieces.)

It would be very helpful for the ARIA WG to know whether these may be supported.

**Comment by pkra** [comment]
It seems the two properties should be supported since JAWS 2024.2406.121 (via https://support.freedomscientific.com/Downloads/JAWS/JAWSWhatsNew, after selecting JAWS 2024). 

However, I was unable to confirm support. E.g., https://codepen.io/pkra/pen/WNgpQbK does not behave as I would have expected it - aria-brailleLabel was ignored on both the image and button.

Is there an example that works? Then I could feed that back into the ARIA 1.3 process.
